omgeo/postprocessors.py

```diff
@@ -271,7 +271,7 @@ def __repr__(self):
 
 class AttrFilter(_PostProcessor):
     """
-    PostProcessor used to ditch results with unwanted attribute values.
+    PostProcessor used to ditch results without desired attribute values.
```

Comment:
How about rephrasing without 'ditch'? Describing what the class does do here for the opening comment would be more useful than this note that was left here about some unclear previous behavior.